lib/activerecord-import/import.rb

```diff
@@ -582,7 +582,8 @@ def find_associated_objects_for_import(associated_objects_by_class, model)
           child.public_send("#{association_reflection.foreign_key}=", model.id)
           # For polymorphic associations
           association_reflection.type.try do |type|
-            child.public_send("#{type}=", model.class.name)
+            model_class_name = model.class.superclass.name == "ActiveRecord::Base" ? model.class.name : model.class.superclass.name
```

Comment:
In Rails 5 ActiveRecord objects extend `ApplicationRecord` instead of `ActiveRecord::Base`.